src/core/transform.js

```diff
@@ -222,8 +222,8 @@ p5.prototype.rotate = function(angle, axis) {
  * 3d box rotating around the x axis.
  */
 p5.prototype.rotateX = function(angle) {
-  p5._validateParameters('rotateX', arguments);
   this._assert3d('rotateX');
+  p5._validateParameters('rotateX', arguments);
```

Comment:
ah this is a little tricky, but when all these pieces are put together, and the `_validateParameters` call is before the `_assert3d` call, then the 'webgl assertions' test will trigger a parameter validation error whereas it's expecting a WEBGL assertion error.

this just switches the order of these so the WEBGL assertion is triggered first and the test then passes.

i think this is the correct way around regardless since whatever arguments you are passing to `rotateX`, if you're not in WEBGL mode then it's an error.

Reply:
got it, thanks!